content/en/03-using-pixie/03-api-quick-start.md

```diff
@@ -188,7 +188,7 @@ script = conn.prepare_script(PXL_SCRIPT)
 
 # Print the table output.
 for row in script.results("http_table"):
-    print(row["http_resp_status"], row["http_req_path"])
+    print(row["http_resp_status"], row["req_path"])
```

Comment:
all of the "http_*" columns are gone now not just req_path. we will need to fully rerun this tutorial to make sure it works.